minet/buzzsumo/types.py

```diff
@@ -0,0 +1,144 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from casanova import TabularRecord
+
+
+@dataclass
+class BuzzsumoArticle(TabularRecord):
+    buzzsumo_id: Optional[str]
+    twitter_shares: Optional[int]
+    facebook_shares: Optional[int]
+    url: Optional[str]
+    updated_at: Optional[str]
+    facebook_likes: Optional[int]
+    total_facebook_shares: Optional[int]
+    title: Optional[str]
+    published_date: Optional[str]
+    pinterest_shares: Optional[int]
+    og_url: Optional[str]
+    article_amplifiers: Optional[str]
+    domain_name: Optional[str]
+    author_name: Optional[str]
+    num_words: Optional[int]
+    subdomain: Optional[str]
+    twitter_user_id: Optional[str]
+    video_length: Optional[str]
+    language: Optional[str]
+    alexa_rank: Optional[str]
+    facebook_comments: Optional[int]
+    num_linking_domains: Optional[int]
+    youtube_views: Optional[int]
+    wow_count: Optional[int]
+    love_count: Optional[int]
+    haha_count: Optional[int]
+    sad_count: Optional[int]
+    angry_count: Optional[int]
+    youtube_likes: Optional[int]
+    youtube_dislikes: Optional[int]
+    youtube_comments: Optional[int]
+    youtube_trending_score: Optional[str]
+    tiktok_download_count: Optional[int]
+    tiktok_play_count: Optional[int]
+    tiktok_share_count: Optional[int]
+    tiktok_comment_count: Optional[int]
+    tiktok_digg_count: Optional[int]
+    tiktok_author: Optional[str]
+    tiktok_music_id: Optional[str]
+    tiktok_music_author: Optional[str]
+    tiktok_music_title: Optional[str]
+    total_shares: Optional[int]
+    thumbnail: Optional[str]
+    total_reddit_engagements: Optional[int]
+    evergreen_score: Optional[str]
+    evergreen_score2: Optional[str]
+    article_types: Optional[List[str]]
+    author_image: Optional[str]
+    ecommerce: Optional[str]
+    is_general_article: bool
+    is_how_to_article: bool
+    is_infographic: bool
+    is_interview: bool
+    is_list: bool
+    is_newsletter: bool
+    is_podcast: bool
+    is_presentation: bool
+    is_press_release: bool
+    is_review: bool
+    is_video: bool
+    is_what_post: bool
+    is_why_post: bool
+
+    @classmethod
+    def parse_booleans(cls, num) -> bool:
```

Comment:
This does not have to be a classmethod. It can remain a helper function at the top of the file.

Reply:
Done.